src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random(System.currentTimeMillis());
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int randomMaxValue = 100;
+    private final int defaultRadius = 10;
+
+    public final Figure getDefaultFigure() {
+        Color defaultColor = Color.WHITE;
+        return new Circle(defaultColor, defaultRadius);
+    }
+
+    public Figure getRandomFigure() {

```

Comment:
Not fixed. Please create private methods for each figure creation.
![image](https://user-images.githubusercontent.com/117547096/210219917-9773ec7f-c955-4a9e-bf09-7e0f06def4b4.png)